src/toil/test/cwl/cwlTest.py

```diff
@@ -703,6 +703,50 @@ def path_with_bogus_rev() -> str:
         except subprocess.CalledProcessError:
             pass
 
+    def test_caching(self) -> None:
+        log.info("Running CWL Test Cache.")
```

Comment:
```suggestion
        log.info("Running CWL caching test.")
```

Also won't Pytest already print what test is being run?